lib/shoulda/matchers/active_model/validate_numericality_of_matcher.rb

```diff
@@ -395,6 +395,11 @@ def is_less_than_or_equal_to(value)
           self
         end
 
+        def is_other_than(value)
```

Comment:
I'm not sure if I really need to fix this issue because it is the gem intended behavior.